modules/kuberay-cluster/main.tf

```diff
@@ -68,7 +68,7 @@ resource "helm_release" "ray-cluster" {
         "iam.gke.io/gke-metadata-server-enabled" : "true"
         } : var.enable_gpu ? {
         "iam.gke.io/gke-metadata-server-enabled" : "true"
-        "cloud.google.com/gke-accelerator" : "nvidia-tesla-t4"
+        "cloud.google.com/gke-accelerator" : "nvidia-l4"
```

Comment:
We also discussed just running the Ray cluster on CPUs right? 

Have we tested if generating the embeddings with CPUs is fast enough?

Reply:
@blackzlq played with this but we haven't benchmarked, next PR